composer.json

```diff
@@ -36,11 +37,15 @@
         "php-http/mock-client": "^0.3.2",
         "phpunit/phpunit": "^5.7 || ^6.0",
         "symfony/process": "^2.3 || ^3.0 || ^4.0",
-        "symfony/http-kernel": "^2.3 || ^3.0 || ^4.0"
+        "symfony/http-kernel": "^2.3 || ^3.0 || ^4.0",
+        "symfony/psr-http-message-bridge": "^1.0",
+        "zendframework/zend-diactoros": "^1.7"
```

Comment:
we already use php-http for other things. lets use the message factory interface from php-http/message instead (see http://docs.php-http.org/en/latest/message/message-factory.html)

Reply:
Good input, done in db97a74d860673d69a6445a73535ffe1ff533164.

Reply:
Ah well, I still need a `ServerRequest` to get a proper Symfony `Request` instance. Not sure I can get rid of Diactoros here, wdyt?